src/ic-cdk/Cargo.toml

```diff
@@ -32,8 +32,12 @@ rstest = "0.12.0"
 
 [features]
 transform-closure = ["dep:slotmap"]
+# The feature below is not additive as normal crate features.
+# Enable it may cause other dependent projects malfunctioning.
+# It is only to be used by canister IDL generation.
+wasi = ["ic0/wasi", "ic-cdk-macros/export_candid"]
```

Comment:
If we are adding a feature, that seems like something that should go in the changelog